### ISSUE TITLE
Show snapshot time?

### ISSUE THREAD

**Issue by Swiftnesses** [twrecked/hass-aarlo#28]
Howdy Mr. Aarlo,

Given the issues I'm having with my old cameras and the snapshot feature, is it possible to somehow show the last snapshot time, perhaps overlaid on the image, or as part of the glance card?

I find myself manually smacking the button and checking for an updated image to ensure the image is actually 'current'...

Many thanks,

Mr. Arlo.

**Comment by twrecked** [comment]
We could do that. We could add it as a title to the image on the screen - either snapshot time for a manually taken one or capture time for a automatically taken one.

**Comment by twrecked** [comment]
Can you try this: https://github.com/twrecked/hass-aarlo/tree/image-title

You might have to refresh your browser a couple of times to get the title to appear!

**Comment by Swiftnesses** [comment]
Refreshed and cleared cache, reboot etc, not seeing anything? 

I'll get my glasses and try again in a little while! :)

**Comment by twrecked** [comment]
Sorry, it'll appear if you hover the mouse over the image.

**Comment by Swiftnesses** [comment]
Argh, sneaky... yes it's working 👍

**Comment by Swiftnesses** [comment]
Don't suppose it's possible to add an overlay somehow to the glance card, I know I'm being pedantic, but I use tablets on the wall and my misses will not be able to 'hover', an overlay would really stop her asking me to check "the bloody camera again as it's not updated!".

**Comment by twrecked** [comment]
Was just about to type... 

I could make it an option to overlay it rather than hover. That would just be a change to the glance component. The important bit is the info is being stored against the camera.

**Comment by twrecked** [comment]
Try now. You will need to add `image_date` to the `show` field on the card config. And you can move things to the top if it's getting crowded. See example following:

```
show:
  - motion
  - snapshot
  - signal_strength
  - captured
  - battery
  - image_date
top_status: true
top_title: true
top_date: true
```

**Comment by Swiftnesses** [comment]
Works a treat! Is there anyway I can format the date in Uk format, i.e. dd-mm?

**Comment by twrecked** [comment]
Yes. Checkout `last_format` under your top `aarlo` config. It uses [strftime](https://www.programiz.com/python-programming/datetime/strftime#format-code) format.

```
aarlo:
 last_format: '%m-%d %H:%M'
```

**Comment by Swiftnesses** [comment]
You're a true Gentleman (missed that setting!), thanks Steve.

**Comment by twrecked** [comment]
Should be fixed by pull request #30.
